core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -81,3 +81,28 @@
     {% endif %}
   </ul>
 </script>
+<style type="text/css">
+.popover{
+  font-size: 13px;
+  max-width: 300px;
+  text-align: center;
+  z-index: -1;
+  color: white;
+  background-color: black;
+}
+.popover.bottom > .arrow{
+  top: -11px;
+  left: 50%;
+  margin-left: -11px;
```

Comment:
If the third party's CSS already has those settings, we should omit them from our code (otherwise it might have unintended consequences)